content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,460 @@ linkTitle: "Workload Manager"
+```
+
+#### Send your certificate's subject to the DIRAC team
+
+In order to configure the DIRAC server so that you get registered as a user, the
+team needs to know your certificate's subject.
+
+Please use the command below on any Unix machine and send its output to Andrei
+Tsaregorodtsev (atsareg`<AT>`in2p3`<DOT>`fr):
+
+```sh
+$ openssl x509 -in $HOME/.globus/usercert.pem -subject -noout
+```
+
+### The EGI Workload Manager Web Portal
+
+To access the EGI Workload Manager open a web browser to:
+[https://dirac.egi.eu/DIRAC/](https://dirac.egi.eu/DIRAC/)
+
+![Fig. 2 - The EGI Workload Manager service Web portal](./figure_2.png)
+
+Fig. 2 - The EGI Workload Manager service Web portal
+
+If you are a new user, you can see the welcome page where you can find links to
+user documentations.  VO options: you can switch to different VOs that you have
+membership.  Log In options: the service supports both X.509, Certificate and
+Check-in log-in.  View options: allow to choose either desktop or tabs layout.
+Menu: a list of tools that enable the selected VO.
+
+#### Upload Proxy
+
+Before submitting your job, you need to upload your Proxy.  Login to the portal.
+Go to:  
+`Menu` > `Tools` > `Proxy Upload`, enter your certificates .p12 file and
+the passphrase, click `Upload`.
+
+![Fig. 3 - The wizard to upload the .p12 proxy certificate](./figure_3.png)
+![Fig. 3.1 - The wizard to upload the .p12 proxy certificate](./figure_3_1.png)
+
+Fig. 3 - The wizard to upload the .p12 proxy certificate
+
+#### Job Submission
+
+Go to:  
+`Menu` > `Tools` > `Job Launchpad`.  First check the `Proxy Status`, click it
+until it shows `Valid` in green color.
+
+In the Job Launchpad, you can select your jobs from the list; add parameters,
+indicating the output Sandbox location.
+
+Now, select `Helloworld` from the  job list, and click `Submit`, you just launch
+your very first job to the EGI HTC cluster.
+
+![Fig. 4 - Submit a job with the Job Launchpad](./figure_4.png)
+
+Fig. 4 - Submit a job with the Job Launchpad
+
+#### Monitor Job status
+
+Go to:  
+`Menu` > `Applications` > `Job Monitor`.  The left panel gives all kinds
+of search options for your jobs.  Set your search criteria, and click ‘Submit’,
+the jobs will list on the right panel.  Try the various options to view
+different information about the jobs.
+
+![Fig. 5 - Monitor the job execution with the Job Monitor panel](./figure_5.png)
+
+Fig. 5 - Monitor the job execution with the Job Monitor panel
+
+#### Get Results from Sandbox
+
+Once the job has been successfully processed, the `Status` of the job will
+change to green.  Right click the job, select:  
+`Sandbox` > `Get Output file(s)`,
+you can get the result file(s).
+
+#### Full User Guide for DIRAC Web Portal
+
+For further instructions, please refer to DIRAC Web Portal Guide
+
+### The DIRAC client tool
+
+The easiest way to install the client is via Docker Container.  If you have a
```

Comment:
A link to docker may help here

Reply:
caba03f881477ddcb6354b5c3dcf39cf2f7dcfae